lib/trento/application/usecases/clusters/clusters.ex

```diff
@@ -60,7 +67,9 @@ defmodule Trento.Clusters do
     query =
       from(c in ClusterReadModel,
         select: c.id,
-        where: c.type == ^ClusterType.hana_scale_up() or c.type == ^ClusterType.hana_scale_out()
+        where:
+          c.type == ^ClusterType.hana_scale_up() or
```

Comment:
This parenthesis look strange.
Shouldn't be?
`(c.type == ^ClusterType.hana_scale_up() or c.type == ^ClusterType.hana_scale_out()) and is_nil(c.deregistered_at)`
Otherwise scale up clusters are always requested, independently of the `deregisterd_at` field

Reply:
Done, nice catch!